DMCompiler/Compiler/DM/DMLexer.cs

```diff
@@ -175,6 +175,7 @@ public partial class DMLexer : TokenLexer {
                                 case "==": token = CreateToken(TokenType.DM_EqualsEquals, c); break;
                                 case "!": token = CreateToken(TokenType.DM_Exclamation, c); break;
                                 case "!=": token = CreateToken(TokenType.DM_ExclamationEquals, c); break;
+                                case "<>": token = CreateToken(TokenType.DM_ExclamationEquals, c); break; // This is syntactically equivalent to the above so why not the same token
```

Comment:
You can use fall-through to avoid writing the same case again.
```suggestion
                                case "<>": // This is syntactically equivalent to the below so why not the same token
                                case "!=": token = CreateToken(TokenType.DM_ExclamationEquals, c); break;
```